### PR TITLE
Add just build-replace-cloudbeat command

### PR DESCRIPTION
**Pull request by amirbenun**
Add `just build-replace-cloudbeat` command that builds a cloudbeat binary and replaces it with the existing cloudbeat on your elastic-agent.

### CODE REVIEW

**Review by oren-zohar — CHANGES_REQUESTED** [review]
pleas add some documentation for the script functions and how to use it

**Review by DaveSys911 — APPROVED** [review]
LGTM for just file

**Comment by ofiriro3** [comment]
Amazing idea!

File: scripts/common.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# The function remotely searches for a file with a given pattern on a given folder
+# $1 is the pod on which it should run
+# $2 is the folder on which it should search
+# $3 is a regex pattern of the file which you want to find
+# The function returns the file name
```
[the remark this answers is not in the document]
Comment (amirbenun) [reply scripts/common.sh]: I have added a bit more explanation, but I don't think an example should help here.
I don't expect users to use this function directly, it's more of a library function being used by other public functions.

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**